web/app/view/ReportController.js

```diff
@@ -423,6 +426,29 @@ Ext.define('Traccar.view.ReportController', {
         this.updateButtons();
     },
 
+    onGeocode: function (positionId) {
+        var position = Ext.getStore('ReportRoute').getById(positionId);
+        if (position && !position.get('address')) {
+            Ext.Ajax.request({
+                scope: this,
+                method: 'GET',
+                url: 'api/server/geocode',
+                params: {
+                    latitude: position.get('latitude'),
+                    longitude: position.get('longitude')
+                },
+                success: function (response) {
+                    position.set('address', response.responseText);
+                    position.commit();
+                    this.fireEvent('selectReport', position);
```

Comment:
Why do we need to fire this event?

Reply:
Clicking on "Show Address" causing selecting the row that causing selecting position and show it in State panel, when comes response address is replaced in Report panel, but not in State.
Looks not very nice. Just updating State panel here.

Reply:
I see. OK.